components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -149,7 +150,10 @@ public class LegacyJobOrchestrator : IJobOrchestrator {
                                     d.DataSet.ExtensionFields["PublisherId"] = jobId;
                                     d.DataSet.ExtensionFields["DataSetWriterId"] = d.DataSetWriterId;
                                 });
+                                var endpoints = string.Join(", ",job.WriterGroup.DataSetWriters.Select(w => w.DataSet.DataSetSource.Connection.Endpoint.Url));
+                                _logger.Information($"Job {jobId} loaded. ConnectionString: {job.ConnectionString} - DataSetWriters endpoints: {endpoints}");
```

Comment:
@oliverchristen, please remove `job.ConnectionString` from the log message. Logging credentials is generally a bit dangerous.

And I am sorry, I missed this the first time I went over the code. Everything else looks good, and I can merge once this is changed. Thanks again for your effort to fix this bug.

Reply:
Oh, you are absolutely right.  Good thing you saw it.

Reply:
Done